pkg/steps/release/promote.go

```diff
@@ -277,7 +277,7 @@ func toPromote(config api.PromotionConfiguration, images []api.ProjectDirectoryI
 // PromotedTags returns the tags that are being promoted for the given ReleaseBuildConfiguration
 func PromotedTags(configuration *api.ReleaseBuildConfiguration) []api.ImageStreamTagReference {
 	var tags []api.ImageStreamTagReference
-	mapping, _ := PromotedTagsWithRequiredImages(configuration, sets.NewString())
+	mapping, _ := PromotedTagsWithRequiredImages(configuration, sets.NewString(), "")
```

Comment:
To avoid passing an empty string, can we instead pass an optional argument as `TagOptions` or something like that?

Reply:
@droslean good idea, done!